scripts/create_perf_json.py

```diff
@@ -1730,6 +1758,10 @@ def to_perf_json(self, outdir: Path):
                 json.dump(events_, perf_json, sort_keys=True, indent=4,
                           separators=(',', ': '))
                 perf_json.write('\n')
+        # Write units and counters data to counter.json file
+        output_counters = Path(outdir, f'counter.json')
```

Comment:
Nit: Replace unnecessary f-string with `'counter.json'`.

Reply:
Updated in new commit